testing/tests/network/basic_connectivity.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// The snarkOS library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkOS library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{common::spawn_test_node_with_nonce, wait_until};
+
+use pea2pea::Pea2Pea;
+use snarkos_testing::{SnarkosNode, TestNode, MAXIMUM_NUMBER_OF_PEERS};
+use tokio::task;
+
+use std::sync::{
+    atomic::{AtomicU8, Ordering::*},
+    Arc,
+};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_nodes_can_connect_to_each_other() {
+    // Start 2 test nodes.
+    let test_node0 = TestNode::default().await;
+    let test_node1 = TestNode::default().await;
+
+    // Ensure that the nodes have no active connections.
+    crate::wait_until!(1, test_node0.node().num_connected() == 0 && test_node1.node().num_connected() == 0);
```

Comment:
```suggestion
    wait_until!(1, test_node0.node().num_connected() == 0 && test_node1.node().num_connected() == 0);
```
It's imported up top ☝️